dev/breeze/src/airflow_breeze/prepare_providers/provider_documentation.py

```diff
@@ -1222,6 +1222,13 @@ def _regenerate_pyproject_toml(context: dict[str, Any], provider_details: Provid
     context["EXTRAS_REQUIREMENTS"] = "\n".join(optional_dependencies)
     context["DEPENDENCY_GROUPS"] = "\n".join(dependency_groups)
 
+    # Most providers require the same python versions, but some may have exclusions
```

Comment:
That's not the right place to set the context values (that's why the CI fails) - here we only override the values that we manually modify in pyproject.toml  (dependencies are manually modified directly in pyproject.toml - so before we regenerate pyproject.toml we read them from there - including comments - so that they are used when we regenerate pyproject.toml with other changes. 

The right place where to set the context is `get_provider_jinja_context` in  `src/airflow_breeze/utils/packages.py`